msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -399,7 +399,10 @@ async def on_teams_channel_restored(
 
 ### Team members added
 
-The `teamMemberAdded` event is sent to your bot the first time it is added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. The user information that is ID, is unique for your bot and can be cached for future use by your service, such as sending a message to a specific user.
+The `teamMemberAdded` event is sent to your bot the first time it's added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. After a team member is added, the bots receives unique channel ID as a conversation ID using the [conversationUpdate](#conversation-update-events) and [installationUpdate](#installation-update-event) events. You can also cache for future use, such as sending a message to a specific user.bots. The bot posts welcome message in the same channel by default instead.
```

Comment:
The team member removed event provides guidance for determining if it's the bot or a user that was removed, but we don't have that info here inthe team member added event. Why?

> To determine if the new member removed was the bot itself or a user, check the Activity object of the turnContext. If the Id field of the MembersRemoved object is the same as the Id field of the Recipient object, then the member removed is the bot, else it is a user. The bot's Id generally is 28:<MicrosoftAppId>.

Reply:
I've updated the per my understanding. @aosolis and @vule-msft request both of you to validate the content: To determine if the new member added was the bot itself or a user, check the `Activity` object of the `turnContext`.  If the `Id` field of the `MembersAdded` object is the same as the `Id` field of the `Recipient` object, then the member added is the bot, else it's a user. The bot's `Id` generally is `28:<MicrosoftAppId>`.